### ISSUE TITLE
Wrong TON 24h and 1h change

### ISSUE THREAD

**Issue by yerofey** [chubin/rate.sx#80]
TON change for 24 hours and for 1 hour is always showing something like 220% change, which is not true.

![Screenshot 2024-05-17 at 14 14 37](https://github.com/chubin/rate.sx/assets/1276038/811e4b3b-0472-4d5f-b002-f8c0ae7dd394)
![Screenshot 2024-05-17 at 14 12 49](https://github.com/chubin/rate.sx/assets/1276038/ab29b22d-917d-49ba-b1d2-e7d9a7bf1503)

**Comment by yerofey** [comment]
Also sparks are not accurate

**Comment by yerofey** [comment]
The `TON` ticker is not found at all, same for `SUI`

<img width="338" alt="Image" src="https://github.com/user-attachments/assets/be994169-d231-4db2-b8e8-c8441c4d7281" />

**Comment by chubin** [comment]
Yes, this is true. The issue with the coin names will be resolved in the near future. As for the incorrect percentage change, it is a bit more challenging to address since that is how the data is provided, but it is possible to fix as well. Accepted.
